django_project/base/views/project.py

```diff
@@ -125,6 +146,27 @@ class ProjectDetailView(ProjectMixin, DetailView):
     template_name = 'project/new_detail.html'
 
     def get_context_data(self, **kwargs):
+        stripe.api_key = settings.STRIPE_KEY
+        ourclientid = settings.OUR_CLIENT_ID
+        myaccountid = settings.MY_ACCOUNT_ID
+        user = self.request.user
+        userid = 0
+        projectid = self.object.id
+        #Get the owner id of the project
+        request = Project.objects.values_list('owner_id',flat=True).get(id=projectid)
+        projectid =request
+        userid = user.id
+        flag=False
+        if userid==projectid:
+            flag=True
+        username = user
+        print("userid: %s" % userid)
```

Comment:
any reason not to use logging? Print is fine while prototyping but should not be in production code.